src/main/webapp/user-page.html

```diff
@@ -17,17 +17,18 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>User Page</title>
     <meta charset="UTF-8">
-    <link rel="stylesheet" href="/css/main.css">
-    <link rel="stylesheet" href="/css/user-page.css">
-    <script src="/js/user-page-loader.js"></script>
```

Comment:
Wait we still need to have `<script src="/js/user-page-loader.js"></script>` and `<link rel="stylesheet" href="/css/user-page.css">` in this file. The js file is the one that helps load all the messages. The css file helps with the styling. I would place these lines back in before we merge into master.

Reply:
got it, changed! thanks for looking out for that